.github/workflows/regenerate-mlir-bindings.yml

```
@@ -22,17 +22,22 @@ jobs:
         run: |
           import Pkg
           Pkg.add("JuliaFormatter")
-      - run: julia --project=. -e 'using Pkg; pkg"instantiate"; pkg"update"; pkg"precompile"; using Clang; Clang.JLLEnvs.get_system_includes()'
+      - name: Instantiate dependencies
+        shell: julia --project=. --color=yes {0}
+        run: |
+          using Pkg
+          pkg"instantiate"
+          pkg"update"
+          pkg"precompile"
```

```
┌ Warning: The Pkg REPL mode is intended for interactive use only, and should not be used from scripts. It is recommended to use the functional API instead.
└ @ Pkg.REPLMode /opt/hostedtoolcache/julia/1.10.5/x64/share/julia/stdlib/v1.10/Pkg/src/REPLMode/REPLMode.jl:382
```
Also i think we dont want to update:
```suggestion
          Pkg.instantiate()
          Pkg.precompile()
```

---

Yeah, I've never agreed with that message. I would argue that a script is an interactive use and that you shouldn't use it in a library which uses `Pkg`.
Anyway, we can change it. No problem.

---

About update... I always call update by default. Update does more than the other commands and it has solved problems I had with the dependencies many times.